src/fx/TexturedGrid.cpp

```diff
@@ -46,11 +47,10 @@ TexturedGrid::TexturedGrid(int width, int height, int gridshift)
 	this->gridHeight = (height>>gridshift)+1;
 
 	grid = new VectorFP[gridWidth*gridHeight];
+	memset(this->grid, 0, (size_t) gridWidth * gridHeight * sizeof(VectorFP));
```

Comment:
Did you get an error for this? Because the grid will be filled in the rendering stage, no need to memset it here. But it doesn't hurt of course.